src/Wizard/WizardContent/WizardContent.tsx

```diff
@@ -3,9 +3,13 @@ import * as React from "react";
 import { Box } from "@mui/material";
 import { WizardContentProps } from "./WizardContent.types";
 
-export default function WizardContent({ children }: WizardContentProps) {
+export default React.forwardRef(function WizardContent(
```

Comment:
![image](https://github.com/IPG-Automotive-UK/react-ui/assets/143519265/85f5f158-b485-4686-b8c3-2a8b32479438)


I think we should add type for ref for type safety
```suggestion
export default React.forwardRef<HTMLDivElement>(function WizardContent(
```